cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
do we need all these comments?

Reply:
Not necessary. These files are created by Cypress. I think it'd help us because it has some specific documentation links in those comments too.